html/base/inputs/Radio.stories.js

```diff
@@ -13,7 +14,11 @@ ${markdownDocumentationLinkBuilder('input')}
   },
 };
 
-export const radioButton = () => `
+export const defaultStory = () => {
```

Comment:
The name will need to be updated in the Angular and React stories as well

Reply:
I'll do it as part of this PR. Commit coming!